spec/student_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Student do
+  # The student should inherit from Person
+  it 'Inherit from Person' do
+    expect(Student.superclass).to eq Person
+  end
+
+  # Instantiate the Student class.
+  before :each do
+    @student = Student.new 20, 2, 'chris'
+  end
```

Comment:
- [ ] Same here. According to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `play_hooky` class method has (at least) an 'it' block 👇🏻 check your student,rb file


```ruby
 # Method play_hooky
  def play_hooky
    "¯\(ツ)/¯"
  end
```

Reply:
You are right, Let's implement it as soon as possible